addons/ui/org.openhab.ui.cometvisu/about_files/ANOTHER-REST-CLIENT_LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Can you refer to a remote location instead, I don't think is available when packaged.